MegaApiClient/Node.cs

```diff
@@ -143,6 +146,13 @@ public void OnDeserialized(StreamingContext ctx)
 
       this.CreationDate = this.SerializedCreationDate.ToDateTime();
 
+      if (string.IsNullOrEmpty(this.SerializedKey))

```

Comment:
Some custom nodes like NodeType.Trash don't have a SerializedKey but are valid. I think you should apply this check to NodeType.File (NodeType.Folder) only

Reply:
Fixed